src/checkbox/styled-components.js

```diff
@@ -143,11 +143,13 @@ function getToggleThumbColor(props) {
 const getToggleCheckMarkStyles = props => {
   const {$checked, $theme, $disabled} = props;
   const {animation, colors} = $theme;
+  const backGroundColor = $disabled ? colors.mono500 : 'white';
```

Comment:
Should probably be consistent here with the "background" camelcase - `backgroundColor`

Reply:
ok